src/dotnet-repl/Repl.cs

```diff
@@ -58,7 +58,7 @@ public class Repl : IDisposable
             {
                 MultiLine = true,
                 Prompt = Theme.Prompt,
-                Highlighter = ReplWordHighlighter.Create()
+                Highlighter = ReplWordHighlighter.Create(_kernel.DefaultKernelName)
```

Comment:
This would now need to react to switching languages during a session.

Reply:
Wait, is it possible? 😅 Okay then I will need to reengineer this a bit. If you have a good hint, please, share!

Reply:
![image](https://user-images.githubusercontent.com/547415/127707277-3369a95e-b9bf-4754-9f3c-3ee606fbd728.png)

Reply:
Oh... right. Then it might be a bit less trivial, thanks, I will see

Reply:
@jonsequitur would it make sense to make Repl mutable? Otherwise we would need to recreate it every time

Reply:
If modifying the `LineEditor` works, then that seems fine.

Reply:
Yep. I also saw that you already have a setter for Theme, so it's anyway mutable XD